src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -57,9 +57,17 @@ const MetaTraderConfig = (() => {
         },
         password_change: {
             title        : localize('Change Password'),
-            success_msg  : response => localize('The main password of account number [_1] has been changed.', [response.echo_req.login]),
+            success_msg  : response => localize('The [_1] password of account number [_2] has been changed.', [response.echo_req.password_type, response.echo_req.login]),
             prerequisites: () => new Promise(resolve => resolve('')),
         },
+        password_reset: {
+            title      : localize('Reset Password'),
+            success_msg: response => localize('The [_1] password of account number [_2] has been changed.', [response.echo_req.password_type, response.echo_req.login]),
```

Comment:
I'm not sure if we need to change the success message, to me `changed` still makes sense in this context.